apps/predictions/lib/repo.ex

```diff
@@ -118,7 +118,8 @@ defmodule Predictions.Repo do
            _schedule_relationship,
            _track,
            _status,
-           _departing?
+           _departing?,
+           _vehicle_id
```

Comment:
thought: I was wondering why this record is a tuple and whether it would be worth updating to a simple struct. But I found [this PR](https://github.com/mbta/dotcom-legacy/pull/1729/files) from 2017 which indicates it is intentional to  reduce memory usage! Perhaps if we are moving towards streaming predictions anyhow this logic will eventually be removed in favor of the `StreamParser`. But both ultimately use the same `Prediction` struct, so making the change to add this field in both places for consistency.